contracts/v2/lib/DepositContractBase.sol

```diff
@@ -89,6 +94,26 @@ contract DepositContractBase {
         assert(false);
     }
 
+    /**
+     * @notice Rollback the merkle tree to a previous state
+     * @param newDepositCount New deposit count (leaf count) to rollback to
+     * @param newFrontier New frontiers (branch array)
+     */
+    function _rollbackTree(
```

Comment:
we could check in smart contract if we want to be a rollback function, and  check that this is the case, or just open it without check anything